### ISSUE TITLE
"Error loading this resource" comes in Firefox

### ISSUE THREAD

**Issue by mangesh** [rtMediaWP/rtMedia#788]
This is a **Firefox** browser specific issue.
**"Error loading this resource"** comes in Firefox even if the media is present and playable.

<img width="615" alt="screen shot 2016-11-23 at 6 41 05 pm" src="https://cloud.githubusercontent.com/assets/246559/20563092/fa860f88-b1ac-11e6-91d4-aa98ad547937.png">

**Comment by raftaar1191** [comment]
The Problem is with Firefox.
I tested by playing video in Firefox version 52.0 but did not get any such error.

Then I downgrade the Firefox version to 46.0.1. Then I was able to regenerate the issue.

The Problem is with poster attribute that is being used video.
Firefox does not support poster attribute before version 47.0 poster attribute supported was being added in version 47.0.

Link: https://github.com/chamilo/chamilo-lms/commit/08a54874ed506d8a50ca7c0c161181947487eb20
